platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/UiSettings.java

```diff
@@ -771,6 +775,27 @@ public void setQuickZoomGesturesEnabled(boolean quickZoomGesturesEnabled) {
     this.quickZoomGesturesEnabled = quickZoomGesturesEnabled;
   }
 
+  /**
+   * Returns zoom gesture rate, including pinch to zoom and quick scale.
+   *
+   * @return The zoom rate.
+   */
+  public float getZoomRate() {
+    return zoomRate;
+  }
+
+  /**
+   * Sets zoom gesture rate, including pinch to zoom and quick scale.
+   * <p>
+   * Default value is 1.0f.
+   * </p>
+   *
+   * @param zoomRate The zoom rate.
+   */
+  public void setZoomRate(float zoomRate) {
```

Comment:
are only positive numbers supported? if yes, can we add something as `@FloatRange(from = 0.0f)`